data/human/culture conversations.txt

```diff
@@ -149,6 +163,49 @@ mission "Paradise Cleaner Incident"
 
 
 
+mission "Curious Waiter"
+	minor
+	source
+		government "Syndicate"
+		attributes "fatory"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`On a whim, you decide to dine in at a restaurant well clear of the busy spaceport. You swiftly find yourself searching up a list of delicacies on a physical menu as people stream past the window by your booth, each looking drained from the long work day. Meal soon ordered, you continue watching the crowd until it arrives, a local version of a katsu Don and dig in.`
```

Comment:
Another spot where people will likely have to Google. I think this one is a bit more redeemable since there wouldn't be any suitable replacement for a specific meal, but curious what others think.

Reply:
My reasoning for the food there for partly because I was craving Japanese food.
My other reason was that for an interstellar species, there's a lack of variety of cultural food.

Reply:
Although I'm not too sure if Katsu Don is capitalised or not, actually, since it's a name of a meal.

Reply:
I'm gonna assume it's not capitalized, based on Google? And since this is a pretty minor mission with the intent of building culture, in hindsight I agree that the specificity of food is appropriate.
```suggestion
			`On a whim, you decide to dine in at a restaurant well clear of the busy spaceport. You swiftly find yourself searching up a list of delicacies on a physical menu as people stream past the window by your booth, each looking drained from the long work day. Meal soon ordered, you continue watching the crowd until it arrives, a local version of a katsu don and dig in.`
```

Reply:
Yep. It's fine.